userapi/src/main/java/fk/prof/userapi/exception/ProfException.java

```diff
@@ -0,0 +1,5 @@
+package fk.prof.userapi.exception;
```

Comment:
+1, although instead of a commons module, we should create specific modules. If we are taking up refactoring right after this PR, I am ok with these changes going in for the time being in interest of getting policy self-serve in production few weeks before bbd

Reply:
Added a TODO [here](https://github.com/Flipkart/fk-prof/commit/d05ec11fd677af720091abf5af0769329510189b), a common module will solve this.